dynamic_programming/count_jump.py

```diff
@@ -0,0 +1,42 @@
+"""
+Given an array of numbers where each element represents the max number of jumps that can be made forward from that element.
+For each array element, count the number of ways jumps can be made from that element to reach the end of the array.
+If an element is 0, then a move cannot be made through that element. The element that cannot reach the end should have a count "-1".
```

Comment:
CONTRIBUTING.md says that max-line-length is 88 characters.

Reply:
Does this algorithm have an actual user or is it just a denial of service attack on the human brain?